src/Maker/MakeStimulusController.php

```diff
@@ -0,0 +1,141 @@
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\WebpackEncoreBundle\WebpackEncoreBundle;
+
+/**
+ * @author Abdelilah Jabri <jbrabdelilah@gmail.com>
+ *
+ * @internal
+ */
+final class MakeStimulusController extends AbstractMaker
+{
+    public static function getCommandName(): string
+    {
+        return 'make:stimulus-controller';
+    }
+
+    public static function getCommandDescription(): string
+    {
+        return 'Creates a new Stimulus controller';
+    }
+
+    public function configureCommand(Command $command, InputConfiguration $inputConfig)
+    {
+        $command
+            ->addArgument('name', InputArgument::REQUIRED, 'The name of the Stimulus controller (e.g. <fg=yellow>hello</>)')
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeStimulusController.txt'));
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        $command->addArgument('extension', InputArgument::OPTIONAL);
+        $command->addArgument('targets', InputArgument::IS_ARRAY, '', []);
+
+        $chosenExtension = $io->choice(
+            'Language (<fg=yellow>JavaScript</> or <fg=yellow>TypeScript</>)',
+            [
+                'js' => 'JavaScript',
+                'ts' => 'TypeScript',
+            ]
+        );
+
+        $input->setArgument('extension', $chosenExtension);
+
+        $includeTargets = $io->confirm('Do you want to include targets?');
+        if ($includeTargets) {
+            $targets = [];
+            $isFirstTarget = true;
+            while (true) {
+                $newTarget = $this->askForNextTarget($io, $targets, $isFirstTarget);
+                $isFirstTarget = false;
+
+                if (null === $newTarget) {
+                    break;
+                }
+
+                $targets[] = $newTarget;
+            }
+
+            $input->setArgument('targets', $targets);
+        }
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    {
+        $controllerName = $input->getArgument('name');
+        $chosenExtension = $input->getArgument('extension');
+        $targets = $input->getArgument('targets');
+
+        $targets = empty($targets) ? $targets : sprintf("['%s']", implode("', '", $targets));
+
+        $fileName = sprintf('%s_controller.%s', $controllerName, $chosenExtension);
+        $filePath = sprintf('assets/controllers/%s', $fileName);
+
+        $generator->generateFile(
+            $filePath,
+            'stimulus/Controller.tpl.php',
+            [
+                'targets' => $targets,
+            ]
+        );
+
+        $generator->writeChanges();
+
+        $this->writeSuccessMessage($io);
+
+        $io->text([
+            'Next:',
+            sprintf('- Open <info>%s</info> and add the code you need', $filePath),
+            'Find the documentation at <fg=yellow>https://github.com/symfony/stimulus-bridge</>',
```

Comment:
Read instead of Find ?
Also perhaps add link to official doc website?

Reply:
Every Maker uses Find not Read:
![image](https://user-images.githubusercontent.com/42073961/155857824-fafc2c61-65c3-4ce2-ae87-4d12094cb79c.png)


@94noni you mean the [Stimulus official doc](https://stimulus.hotwired.dev/handbook/introduction)? or [Symfony UX](https://symfony.com/doc/current/frontend/ux.html#stimulus-tools-around-the-world)?

Reply:
I was talking of the official hotwired stimulus doc

But i am unsure if it is ok and perhaps only symfony doc url is wanted (need core maintainer review :) )

And ok for find